lib/onebox/engine/pdf_onebox.rb

```diff
@@ -0,0 +1,39 @@
+require 'pdf-reader'
+
+module Onebox
+  module Engine
+    class PdfOnebox
+      include Engine
+      include LayoutSupport
+
+      matches_regexp(/^(https?:)?\/\/.*\.pdf(\?.*)?$/i)
+      always_https
+
+      private
+
+      def data
+        html_entities = HTMLEntities.new
+        pdf_response = get_pdf_response
+        pdf_data = pdf_response.info
```

Comment:
Currently the oneboxing fails in that case. Should we onebox the pdf link anyway even if `pdf-reader` is unable to fetch info for that pdf file?